### ISSUE TITLE
DateTime() field gets updated to Null if not set in an update query

### ISSUE THREAD

**Issue by remy-beaumont-moonshothealth** [octabytes/FireO#63]
Given the following model:

```
class User(Model):
    user_id = IDField()
    name = TextField()
    date_added = DateTime()
```

A user is created with :
```
User.collection.create(user_id='1',name='Person Name', date_added=datetime.now())
```

His name is then changed, but the date_added is not touched:
```
user = User(name='New name')
user.update(key='1')
```

After this update, the date_added value is changed to Null in Firestore.

Looking at the (update_query.py)[https://github.com/octabytes/FireO/blob/master/src/fireo/queries/update_query.py], we can see why it's happening. If the value of the DateTime field is not set, it's value is returned.
It also seems like bool is also always returned even if it's value is not changed.

```
    for f in self.model._meta.field_list.values():
            # Check if it is nested model
            if isinstance(f, NestedModel):
                # Get nested model field
                self._nested_field_list(f, field_dict, f.name)
            else:
                v = f.get_value(self.query.get(f.name), ignore_required=True, ignore_default=True)
                if v is not None or type(v) is bool:
                    field_dict[f.db_column_name] = v
                if v is None and isinstance(f, DateTime):
                    field_dict[f.db_column_name] = v
        return field_dict
```

Is there a reason for this ?

Thank you,
Rémy 

(BTW, really like FireO)

**Comment by AxeemHaider** [comment]
issue is fixed in version `v1.3.1`

**Comment by remy-beaumont-moonshothealth** [comment]
That was quick !
